datasets/hls-events.ej.data.mdx

```diff
@@ -29,7 +29,7 @@ layers:
       - 4
       - 20
     sourceParams:
-      post_process: swir
+      algorithm: swir
```

Comment:
if we want zero downtime, we can probably put `post_process` and `algorithm` both for now and get rid of `post_process` later?

Reply:
yeah that works, too, thanks for pointing it out! https://test-raster.delta-backend.com/mosaic/1b7cfa7ab6b9f8890a7f52d824b4e17c/tiles/WebMercatorQuad/8/81/114?assets=B07&assets=B05&assets=B04&algorithm=swir&post_process=swir

Reply:
OH actually no we can't use the algorithm param until after the upgrade
https://staging-raster.delta-backend.com/mosaic/tiles/1b7cfa7ab6b9f8890a7f52d824b4e17c/WebMercatorQuad/9/162/228@1x?assets=B07&assets=B05&assets=B04&post_process=swir&algorithm=swir
```
{
  "detail": [
    {
      "loc": [
        "query",
        "algorithm"
      ],
      "msg": "unexpected value; permitted: 'hillshade', 'contours', 'normalizedIndex', 'terrarium', 'terrainrgb'",
      "type": "value_error.const",
      "ctx": {
        "given": "swir",
        "permitted": [
          "hillshade",
          "contours",
          "normalizedIndex",
          "terrarium",
          "terrainrgb"
        ]
      }
    }
  ]
}
```

Reply:
Ah - ok then we should time the deploy!